.circleci/config.yml

```diff
@@ -544,6 +544,21 @@ jobs:
             wasmfs.test_readdir_unlink
             wasmfs.test_unistd_pipe
             wasmfs.test_fs_write
+            wasmfs.test_fs_writev
+            wasmfs.test_fs_writev_rawfs
+            wasmfs.test_fs_writeFile
+            wasmfs.test_fs_writeFile_rawfs
+            wasmfs.test_fs_readv
+            wasmfs.test_fs_write
+            wasmfs.test_fs_readv_rawfs
+            wasmfs.test_fs_nodefs_nofollow
+            wasmfs.test_fs_nodefs_readdir
+            wasmfs.test_fs_nodefs_home
+            wasmfs.test_fs_nodefs_cloexec
+            wasmfs.test_fs_nodefs_cloexec_rawfs
+            wasmfs.test_fs_errorstack
+            wasmfs.test_fs_errorstack_rawfs
+            wasmfs.test_fs_emptyPath
```

Comment:
Should we just put ` wasmfs.test_fs_*` here?

Reply:
Not all the `test_fs_*` tests are passing yet, since some are failing because the functions haven't been implemented yet.